packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -19,18 +19,19 @@ import { ConsoleLogger, InstrumentationLibrary } from '@opentelemetry/core';
 import { Resource } from '@opentelemetry/resources';
 import { BatchObserverMetric } from './BatchObserverMetric';
 import { BaseBoundInstrument } from './BoundInstrument';
-import { MetricKind } from './export/types';
-import { UpDownCounterMetric } from './UpDownCounterMetric';
 import { CounterMetric } from './CounterMetric';
-import { UpDownSumObserverMetric } from './UpDownSumObserverMetric';
-import { ValueRecorderMetric } from './ValueRecorderMetric';
-import { Metric } from './Metric';
-import { ValueObserverMetric } from './ValueObserverMetric';
-import { SumObserverMetric } from './SumObserverMetric';
-import { DEFAULT_METRIC_OPTIONS, DEFAULT_CONFIG, MeterConfig } from './types';
 import { Batcher, UngroupedBatcher } from './export/Batcher';
 import { PushController } from './export/Controller';
 import { NoopExporter } from './export/NoopExporter';
+import { MetricKind } from './export/types';
+import { Metric } from './Metric';
+import { SumObserverMetric } from './SumObserverMetric';
+import { DEFAULT_CONFIG, DEFAULT_METRIC_OPTIONS, MeterConfig } from './types';
+import { UpDownCounterMetric } from './UpDownCounterMetric';
+import { UpDownSumObserverMetric } from './UpDownSumObserverMetric';
+import { ValueObserverMetric } from './ValueObserverMetric';
+import { ValueRecorderMetric } from './ValueRecorderMetric';
+import merge = require('lodash.merge');
```

Comment:
organized imports.

N.B. import merge = require('lodash.merge'); because to import from would require esModuleInterop to be true due to the nature of lodash single function exports.